tests/previews.ml

```diff
@@ -0,0 +1,32 @@
+open Lwt
+module D = Dropbox_lwt_unix
+
+let download t fn =
+  D.previews t fn >>= function
+  | None -> Lwt_io.printlf "No file named %S." fn
+  | Some(c_t, c_l, stream) ->
+    let format = match c_t with
+      | "application/pdf" -> ".pdf"
+      | _ -> ".html" in
+    let fn = Filename.basename fn in
+    let fname = "preview_of_" ^ (Filename.chop_extension fn) ^ format in
```

Comment:
Like thumbnails function, Dropbox doesn't accept file without extension and return an Error: Invalid_arg {"error":"Previews not available for files with the extension \"\"."} .